Quora/templates/index.html

```diff
@@ -97,4 +97,11 @@
         </div>
     </div>
 
+    {% for post in list %}
+        <p>"id" {{ post.id}}</p>
+        <p>"question:"{{ post.question }}</p>
+        <p>"date":{{ post.post_date }}</p>
+        <p>"user":{{ post.user }}</p>
+    {% endfor %}
```

Comment:
This part should be deleted and instead redirect to the template view_question.html with the corresponding content.

Btw, user is None because there isn't a user field in the template. It should be extracted from the user logged that creates the post.